docs/getting-started-guides/README.md

```diff
@@ -28,38 +28,38 @@ For the easiest "kick the tires" experience, please try the [local docker](docke
-Bare-metal           | custom       | CentOS | _none_      | [docs](centos/centos_manual_config.md)       |             | Community(@coolsvap)         | Uses K8s v0.9.1
-AWS                  | Juju         | Ubuntu | flannel     | [docs](juju.md)                              |             | [Community](https://github.com/whitmo/bundle-kubernetes) ( [@whit](https://github.com/whitmo), [@matt](https://github.com/mbruzek), [@chuck](https://github.com/chuckbutler) ) | [Tested](http://reports.vapour.ws/charm-tests-by-charm/kubernetes) K8s v0.8.1
-OpenStack/HPCloud    | Juju         | Ubuntu | flannel     | [docs](juju.md)                              |             | [Community](https://github.com/whitmo/bundle-kubernetes) ( [@whit](https://github.com/whitmo), [@matt](https://github.com/mbruzek), [@chuck](https://github.com/chuckbutler) ) | [Tested](http://reports.vapour.ws/charm-tests-by-charm/kubernetes) K8s v0.8.1
-Joyent               | Juju         | Ubuntu | flannel     | [docs](juju.md)                              |             | [Community](https://github.com/whitmo/bundle-kubernetes) ( [@whit](https://github.com/whitmo), [@matt](https://github.com/mbruzek), [@chuck](https://github.com/chuckbutler) ) | [Tested](http://reports.vapour.ws/charm-tests-by-charm/kubernetes) K8s v0.8.1
-AWS                  | Saltstack    | Ubuntu | OVS         | [docs](aws.md)                               |             | Community (@justinsb)        | Uses K8s version 0.5.0
-Vmware               | CoreOS       | CoreOS | flannel     | [docs](coreos.md)                            |             | Community (@kelseyhightower) | Uses K8s version 0.15.0
-Azure                | Saltstack    | Ubuntu | OpenVPN     | [docs](azure.md)                             |             | Community                    |
-Bare-metal           | custom       | Ubuntu | flannel     | [docs](ubuntu.md)                            |             | Community (@resouer @WIZARD-CXY)       | use k8s version 0.19.3
-Local                |              |        | _none_      | [docs](locally.md)                           |             | Community (@preillyme)      |
-libvirt/KVM          | CoreOS       | CoreOS | libvirt/KVM | [docs](libvirt-coreos.md)                    |             | Community (@lhuard1A)       |
-oVirt                |              |        |             | [docs](ovirt.md)                             |             | Community (@simon3z)        |
-Rackspace            | CoreOS       | CoreOS | flannel     | [docs](rackspace.md)                         |             | Community (@doublerr)       | use k8s version 0.18.0
+IaaS Provider        | Config. Mgmt | OS     | Networking  | Docs                                              | Conforms | Support Level
+-------------------- | ------------ | ------ | ----------  | ---------------------------------------------     | ---------| ----------------------------
```

Comment:
I am not.  People can put this in the getting-started-guide if they really want to.

This was useful when kubernetes was changing rapidly.
However, it was not updated reliably.  Also, simply using
a particular binary version does not assure users that
the distribution provides a conformant implementation.
The conformance test does provide that, and I'm hoping
that more distros will use that in the future.
Therefore, I've deleted the Notes column in favor of the
Conforms column.